s3/src/main/scala/akka/stream/alpakka/s3/impl/S3Stream.scala

```diff
@@ -148,9 +162,13 @@ private[alpakka] final class S3Stream(credentials: AWSCredentials, region: Strin
     case s => Some(Paths.get(s))
   }
 
-  private def chunkAndRequest(s3Location: S3Location, chunkSize: Int = MinChunkSize)(
-      parallelism: Int = 4): Flow[ByteString, UploadPartResponse, NotUsed] =
-    createRequests(s3Location, chunkSize, parallelism).via(Http().superPool[(MultipartUpload, Int)]()).map {
+  private def chunkAndRequest(
+      s3Location: S3Location,
+      contentType: ContentType,
+      cannedAcl: CannedAcl = CannedAcl.Private,
+      chunkSize: Int = MinChunkSize)(parallelism: Int = 4): Flow[ByteString, UploadPartResponse, NotUsed] =
+    createRequests(s3Location, contentType, cannedAcl, chunkSize,
+      parallelism).via(Http().superPool[(MultipartUpload, Int)]()).map {
```

Comment:
that is one weird linefeed, make it after `)` before `.via` instead, and then another before `.map`